docs/reference/composites.rst

```diff
@@ -258,3 +258,69 @@ Methods
    VirtualGraphComposite.sample
    VirtualGraphComposite.sample_ising
    VirtualGraphComposite.sample_qubo
+
+Reverse
+=======
+
+Composites that do batch reverse annealing. Can accept batch initial states or a set
+of schedules.
```

Comment:
How about something like "Composites that batch reverse-annealing submissions based on sets of initial states or anneal schedules"?